shared/estate.js

```diff
@@ -21,14 +21,18 @@ export function toEstateObject(estatesArray) {
     )
 }
 
-export function calculateZoomAndCenter(parcels) {
+export function calculateMapProps(parcels, size = 20) {
```

Comment:
`pan` and `zoom` are part of the [ParcelCanvas state](https://github.com/decentraland/marketplace/blob/master/webapp/src/components/ParcelPreview/ParcelCanvas/ParcelCanvas.js#L88-L92), what I did on this PR was making the `panX` and `panY` configurable from prop (it wasn't like that before because we never had a use case for it). 

I used `pan` and `zoom`  because those are names is in most map libraries like [Google Maps](https://developers.google.com/maps/documentation/javascript/interaction) but I could send a commit to document the [ParcelCanvas API](https://github.com/decentraland/marketplace/blob/master/webapp/src/components/ParcelPreview/ParcelCanvas/ParcelCanvas.js#L41-L59) to describe these props if that makes it more clear? I think that would be a better way to document it than in this helper since this is only used to pass it down to `<ParcelCanvas>`

Reply:
Thanks!

I think it could be documented a little bit, as I am not familiar with the Google Maps API  

Could be done in another PR if you think this one is not the best place.

Reply:
I just added a commit (df576a) documenting the ParcelCanvas API

Reply:
Amazing!